bin/cli.js

```diff
@@ -31,28 +55,26 @@ Object.keys(fileConfig).forEach(configPropName => {
   }
 });
 
-if (!url || flags.help) {
-  if (!flags.help) console.error(getMessageWithPrefix('ERROR', 'NO_URL'));
-  console.error('Usage:');
-  console.error('    pwmetrics http://example.com/');
-  console.error('    pwmetrics http://example.com/ --json  Reports json details to stdout.');
-  console.error('    pwmetrics http://example.com/ --runs=n  Does n runs (eg. 3, 5), and reports the median run\'s numbers.');
-  console.error('    pwmetrics --expectations  Expectations from metrics results. Useful for CI. Config required.');
-  console.error('    pwmetrics --submit  Submit metric results into sheets. Config required.');
-  console.error('    pwmetrics --config  Read config form file.');
-
-  return;
-}
+const write = function(fileName, data) {
+  const path = sysPath.join(process.cwd(), fileName);
+  fs.writeFile(path, data, err => {
+    if (err) throw err;
 
+    console.log(getMessageWithPrefix('SUCCESS', 'SAVED_TO_JSON', path));
+    process.exit(0);
+  });
+};
 
 const pwMetrics = new PWMetrics(url, config);
 pwMetrics.start()
   .then(data => {
-    if (flags.json) {
-      data = JSON.stringify(data, null, 2) + '\n';
+    data = JSON.stringify(data, null, 2) + '\n';
+    if (cliFlags.json && !cliFlags.json.length) {
       data && process.stdout.write(data);
+      process.exit(0);
+    } else if (cliFlags.json.length) {
```

Comment:
If no ```json``` flag is used this line will throw an error like this:
```
$node bin/cli.js localhost:4000

... //Graph data

TypeError: Cannot read property 'length' of undefined
    at pwMetrics.start.then.data (/Users/pedrocardoso.silva/dev/pwmetrics/bin/cli.js:75:29)
```

Perhaps wrap the different JSON outputs all inside an if(cliFlags.json)